helix-term/src/args.rs

```diff
@@ -1,12 +1,24 @@
 use anyhow::{Error, Result};
+use helix_view::editor::File;
 use std::path::PathBuf;
 
 #[derive(Default)]
 pub struct Args {
     pub display_help: bool,
     pub display_version: bool,
     pub verbosity: u64,
-    pub files: Vec<PathBuf>,
+    pub files: Files,
+}
+
+pub enum Files {
```

Comment:
I don't like this extra enum.

Reply:
What alternative do you think there is? I don't know if there is a better way.